test/core/reporters/test_coverage/test_coverage_cob_reporter.pkb

```diff
@@ -0,0 +1,44 @@
+create or replace package body test_coverage_cob_reporter is
+
+  procedure report_on_file is
+    l_results   ut3.ut_varchar2_list;
+    l_expected  clob;
+    l_actual    clob;
+  begin
+    --Arrange
+    l_expected := 
+    '<?xml version="1.0"?>
+<!DOCTYPE coverage SYSTEM "http://cobertura.sourceforge.net/xml/coverage-04.dtd">
+<coverage line-rate="0" branch-rate="0.0" lines-covered="2" lines-valid="3" branches-covered="0" branches-valid="0" complexity="0" version="1" timestamp="%"><sources>
+<source>test/ut3.dummy_coverage.pkb</source>
+</sources>
+<packages>
+<package name="test/ut3.dummy_coverage.pkb" line-rate="0.0" branch-rate="0.0" complexity="0.0">
```

Comment:
Shouldn't the package name be DB object name `ut3.dummy_coverage` rather than the file name?